agent/installer/installer.go

```diff
@@ -24,6 +26,8 @@ const (
 	ErrBundleUninstall   = Error("Error uninstalling bundle")
 )
 
+var PreRequisitePackages = []string{"socat", "ebtables", "ethtool", "conntrack"}
```

Comment:
oohhh.. I see this is public so that we can override to test it? I'm personally not a fan. I would rather keep this private - since we don't really want consumers to be able to override this. Right? 

If we do, then I would rather inject this via a constructor. 

For testing, this might be hard (and maybe even pointless) to unit test. Might be easier to write an integration / e2e test of sorts (something that runs in a container).

Reply:
The e2e test already checks this.
So I will make it private and remove the unit tests.